src/DocTest/Ast.elm

```diff
@@ -0,0 +1,137 @@
+module DocTest.Ast exposing (..)
+
+
+type alias TestSuite =
+    { imports : List String
+    , tests : List Test
+    , functionName : Maybe String
+    , functions : List Function
```

Comment:
What the functions inside a test suite? How do they relate to the `functionName`?